src/main/java/org/betonquest/betonquest/compatibility/citizens/CitizensVariable.java

```diff
@@ -7,65 +7,78 @@
 import org.betonquest.betonquest.api.Variable;
 import org.betonquest.betonquest.api.profiles.Profile;
 import org.betonquest.betonquest.exceptions.InstructionParseException;
-import org.bukkit.Location;
+import org.betonquest.betonquest.variables.LocationVariable;
 
+import java.util.Arrays;
 import java.util.Locale;
 
 /**
  * Provides information about a citizen npc.
  * <p>
  * Format:
- * {@code %citizen.<id>.<type>%}
+ * {@code %citizen.<id>.<argument>.<mode>.<precision>%}
  * <p>
- * Types:
+ * Arguments:
  * * name - Return citizen name
  * * full_name - Full Citizen name
- * * location  - Return citizen location. x;y;z;world;yaw;pitch
+ * * location - Return citizen location, defaults to ulfLong
+ * Modes:
+ * * xyz - The x, y and z location of the npc, separated by spaces
+ * * x - The x location of the npc
+ * * y - The y location of the npc
+ * * z - The z location of the npc
+ * * world - The world location of the npc
+ * * yaw - The yaw of the npc
+ * * pitch - The pitch of the npc
+ * * ulfShort - The location of the npc in the form x;y;z;world
+ * * ulfLong - The location of the npc in the form x;y;z;world;yaw;pitch
+ * Precision is decimals of precision desired, defaults to 0.
```

Comment:
This should only link to the javadocs of `LocationVariable` and should not repeat the implementation of the other class

Reply:
LocationVariable has no javadoc. Are you sure this is what you want?

Reply:
yes, thats then the issue of the LocationVariable. But as you already wrote some docs you can also move them to the LocationVariable

Reply:
also, main does not verify right now... before my changes?

Reply:
Updated.